documentdb/spring-data-azure-documentdb/src/test/java/com/microsoft/azure/spring/data/documentdb/repository/core/DocumentDbTemplateIT.java

```diff
@@ -16,25 +16,31 @@
 import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.domain.EntityScanner;
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.PropertySource;
 import org.springframework.data.annotation.Persistent;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 
 import java.util.List;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
+@RunWith(SpringJUnit4ClassRunner.class)
 @PropertySource(value = {"classpath:application.properties"})
 public class DocumentDbTemplateIT {
 
-    private static final String DOCUMENTDB_URL_PROPERTY_NAME = "documentdb.uri";
-    private static final String DOCUMENTDB_KEY_PROPERTY_NAME = "documentdb.key";
     private static final String TEST_DB_NAME = "testdb";
     private static final Person TEST_PERSON = new Person("testid", "testfirstname", "testlastname");
+    
+    @Value("${documentdb.uri}")
```

Comment:
Can you try @Value("${DOCUMENTDB_URI}")? I think it works and the application.properties file won't be needed.